### PR TITLE
Add icon

### PR DESCRIPTION
**Pull request by dash102**

### CODE REVIEW

**Comment by scottmc** [comment]
Can you export the file in IOM as an .rdef file?  Then create a proper .rdef file for haiku2048, adding the icon array to it?  See https://github.com/scottmc/BurnItNow/blob/master/burnitnow2/Source/BurnItNow.rdef
for an example of the .rdef file.

**Comment by dash102** [comment]
The link isn't working for me :/

**Comment by scottmc** [comment]
Sorry about that.  I just deleted that old repo and right afterwards remembered that I linked to it here.  
It's now at:
https://github.com/HaikuArchives/BurnItNow/blob/master/Source/BurnItNow.rdef

**Comment by dash102** [comment]
Where do I get the information for:
```
resource app_signature "application/x-vnd.haikuarchives.BurnItNow2";

resource app_flags B_EXCLUSIVE_LAUNCH;

resource app_version {
	major  = 0,
	middle = 1,
	minor  = 1,

	variety = B_APPV_DEVELOPMENT,
	internal = 0,

	short_info = "",
	long_info = ""
};

resource file_types message;


resource vector_icon array {
```

for the 2048 app?

**Comment by scottmc** [comment]
I'd go with:
resource app_signature "application/x-vnd.twofx.haiku2048";

for app_version set them all to 0, we'll let txofx decide when he want to set a version for it.
the rest you can keep the same, cept for the vector_icon array which will be the .rdef you exported from IOM.

**Comment by dash102** [comment]
Sorry, finals had me pretty busy the past few days. Do I put the .rdef in the same artwork/ directory as the iom file?

**Comment by scottmc** [comment]
yes, you can put the  i-o-m exported icon .rdef file into the artwork folder.  But you should copy the vector array data and paste it into the .rdef file in the main source directory in the section:

resource vector_icon array {
put array data here
}

Then edit the makefile to add the main rdef file to the rdefs section as such:
RDEFS = haiku2048.rdef

**Comment by dash102** [comment]
Thanks for the help - I didn't realize that "resource vector_icon array" wasn't supposed to be replaced by the name of the icon. Make works now :)

File: haiku2048.rdef [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2015 Markus Himmel
+ * Distributed under the terms of the MIT License
+ * Haiku Icon by Stephanie Fu
+ */
+
+resource app_signature "application/x-vnd.twofx.haiku2048";
```
Comment (TwoFX) [inline haiku2048.rdef]: The app signature of this program is declared as `application/x-vnd.Haiku-Haiku2048` in App.cpp. One of the two or both should be changed so they match; I don't know what naming conventions there are for app signatures, they should be changed to something idiomatic.

**Comment by scottmc** [comment]
Not sure why this was still pending.  Going to merge it now.

**Pull request merged**